source/graphics/GridGraphic.js

```diff
@@ -146,6 +146,13 @@ export default class GridGraphic extends Graphic {
     return this._tiles
   }
 
+  importTiles(tiles) {
+    const maxX = Math.max.apply(null, tiles.map(tile => tile.position.x))
```

Comment:
isn't there something you can do with the `...` rest arguments here instead of `apply` I thought there was an eslint warning about that but maybe doesn't apply :P here?